conans/client/loader_txt.py

```diff
@@ -8,7 +7,7 @@ class ConanFileTextLoader(object):
     def __init__(self, input_text):
         # Prefer composition over inheritance, the __getattr__ was breaking things
         self._config_parser = ConfigParser(input_text,  ["requires", "generators", "options",
-                                                         "imports", "tool_requires"],
+                                                         "tool_requires"],
```

Comment:
Maybe don't remove this one, so parsing doesn't crash for 1.X-2 compatible conanfile.txt?